frontend/src/app/modules/availability/new-slot/new-availability-page/new-availability-page.component.ts

```diff
@@ -64,7 +64,7 @@ export class NewAvailabilityPageComponent extends BaseComponent {
             questions: this.newAvailabilityComponent.questionsComponent.questions,
             schedule: this.newAvailabilityComponent.scheduleComponent.schedule,
             templateSettings: this.newAvailabilityComponent.notificationEmailsComponent.settings,
-            teamId: this.currentTeamId,
+            teamId: this.currentTeamId ?? 3,
```

Comment:
Please check what you commit

Reply:
my bad. Hardcoded for avoiding annoying error of teams